politeiawww/dcrdata/wsdcrdata.go

```diff
@@ -112,54 +112,57 @@ func (w *wsDcrdata) subscribe(event string) error {
 	}
 
 	w.addSub(event)
-	log.Debugf("wsDcrdata successfully subscribed to %v", event)
+	log.Debugf("WSDcrdata successfully subscribed to %v", event)
 	return nil
 }
 
 // unsubscribe ubsubscribes the dcrdata client from an event.
-func (w *wsDcrdata) unsubscribe(event string) error {
-	if w.isShutdown() {
-		return errShutdown
+func (w *WSDcrdata) unsubscribe(event string) error {
```

Comment:
Should `unsubscribe` and `ping` also be exported?

Reply:
Unsubscribe should. I'm not sure if anyone would use Ping from outside the package, but might as well export too.